src/usdb_syncer/notes_parser.py

```diff
@@ -85,8 +85,8 @@ class LineBreak:
     the next line start.
     """
 
-    start: int
-    end: int | None
+    out_time: int
+    in_time: int | None
```

Comment:
I find this naming less intuitive. I would expect attributes to refer to the class they belong to, in this case out time to be the end of the line break. However, I admit the previous names weren't ideal, either.
Any other ideas? Maybe `last_line_out_time` and `next_line_in_time`?

Reply:
The new naming came from UltraStar Manager and I find it more understandable than start/end, but possibly still not ideal. I am okay with your suggestion, even though then it should be `current_line_out_time` and `next_line_in_time`, right?

Reply:
You mean because `LineBreak` is part of a `Line`? I'd have said that's more of an implementation detail, but either works for me.
Maybe strictly speaking, each line should have an `in_time` and `out_time` instead. But this might be more work than it's worth.

Reply:
Yeah, true. From the point of view of the line_break, looking back it `last_line_out_time` (or `previous_line_out_time`) and looking forward it’s `next_line_in_time`. Either one is fine for me as well. I get your last point and agree that it’s more work than it’s worth.